packages/core/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "UMA smart contracts and unit tests",
   "dependencies": {
     "@gnosis.pm/safe-contracts": "^1.3.0",
-    "@gnosis.pm/zodiac": "1.0.3",
+    "@gnosis.pm/zodiac": "3.2.0",
```

Comment:
No contract changes necessary?

Reply:
Upgrade does not change interfaces for anything that Optimistic Governor is using. Double checked all interaction points with external dependencies:
- `Enum.Operation` for type of operation is from `@gnosis.pm/safe-contracts` that keeps the same version
- `initializer() modifier from `@openzeppelin/contracts-upgradeable` has changed some of its internal logic and added `Initialized` event, but this should not affect how Optimistic Governor is set up.
- `__Ownable_init()` from `@openzeppelin/contracts-upgradeable` is the same interface and implementation
- onlyOwner() modifier from `@openzeppelin/contracts-upgradeable` only has moved its require inside internal method, but the logic is the same
- `exec` from `@gnosis.pm/zodiac` has the same interface. Its logic is patched on how transaction guard is checked, but that does not affect Optimistic Governor contract directly